README.md

```diff
@@ -117,46 +118,41 @@ A simple wrapper for the Jaeger exporter of [opentelemetry-java](https://github.
 | ------------------------------------ | ----------------------------------   | -------------- | -------------------------------------------------------- |
 | splunk.context.server-timing.enabled | SPLUNK_CONTEXT_SERVER_TIMING_ENABLED | false          | Enables adding `Server-Timing` header to HTTP responses. |
 
-### More options
-For more options see [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation#configuration-parameters-subject-to-change)
-
-## Span tags the Splunk distribution automatically creates
```

Comment:
I vaguely remember somebody wanted to see all the attributes that our distribution adds on top of the standard Otel...

Reply:
Maybe we can move them to a separate doc and reference it from the main README?